src/utils/jsonToHtml.ts

```diff
@@ -1,8 +1,9 @@
 let level = 1;
-const COLLAPSE_LEVEL = 2;
+let collapseLevel;
```

Comment:
Get rid of this global variable. Use `maxCollapseLevel` in functions `arrayToHTML` and `objectToHTML`. They have the `maxCollapseLevel` in scope.

Reply:
Should I provide `maxCollapseLevel` via arguments into these functions?